README.md

```diff
@@ -33,8 +33,12 @@ ReactDOM.render((
 ), document.getElementById('react-root'));
 ```
 
-Define an object containing your queries that a particular `Relay.Container`
-needs and add it as a `queries` prop to any container `<Route/>`s:
+`react-router-relay` will automatically generate a component that includes all of your fragments, and a route that includes all of your root queries,
+and dispatch/render everything in one go.
+
+# Queries
+
+`react-router`’s `Route` already has a name and params: this means that all we need to constuct a full `Relay.Route` is `queries` property, which we pass on to `Route` as a prop:
```

Comment:
R-R 1.0.0 `Route`s don't have names - we auto-generate the name from the queries used (we need to uniquely and stably identify those queries anyway to get the mapping to work).